packages/core/src/redocly/index.ts

```diff
@@ -160,16 +162,15 @@ export class RedoclyClient {
 
 export function isRedoclyRegistryURL(link: string): boolean {
   const domain = process.env.REDOCLY_DOMAIN || DOMAINS[DEFAULT_REGION];
-  if (!link.startsWith(`https://api.${domain}/registry/`)) return false;
-  const registryPath = link.replace(`https://api.${domain}/registry/`, '');
 
-  const pathParts = registryPath.split('/');
+  const legacyDomain = domain === 'redocly.com' ? 'redoc.ly' : domain;
```

Comment:
IMHO, it's a bit hard to understand logic here.

Reply:
I think it'll be more simple if you just hardcode 'redoc.ly' (you're comparing it with hardcoded values btw)

Reply:
>  if you just hardcode 'redoc.ly' 

In such case someone on EU region can get `redoc.ly/*` links matching too but it should not.